target/linux/lantiq/dts/EASY220.dtsi

```diff
@@ -0,0 +1,295 @@
+#include "vr9.dtsi"
+#include <dt-bindings/input/input.h>
+
+/ {
+
+	compatible = "lantiq,xway", "lantiq,vr9", "lantiq,xrx220";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power_g;
+		led-failsafe = &power_a;
+		led-running = &power_g;
+
+		led-dsl = &dsl;
+		led-internet = &lan_g;
+	};
+
+	memory@0 {
+		reg = <0x0 0x3f00000>;
+	};
+
+	ltq_swreset {
+		compatible = "lantiq,ltq_swreset";
+		swreset_pin = <&gpio 7 GPIO_ACTIVE_HIGH>;
+		swreset_bit = <1>;
+		status = "ok";
+	};
+
+	gphy-xrx200 {
+		compatible = "lantiq,phy-xrx200";
+		firmware = "lantiq/vr9_phy22f_a2x.bin";
+		phys = [ 00 01 ];
+	};
+
+//	gpio-keys-polled {
```

Comment:
What is wrong with the reset key? Why is it comment. We usually do not keep disabled code in the dts.